package.json

```diff
@@ -4,23 +4,121 @@
   "description": "Lambda Function to Stream Logs from AWS S3 to LogDNA",
   "main": "index.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint -c .eslintrc *.js"
+    "docs": "config-doc lib/config.js > doc/env.md",
+    "lint": "eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "tap": "tap",
+    "pretest": "npm run lint",
+    "pretest:ci": "npm run lint",
+    "test": "npm run tap",
+    "test:ci": "tools/test-ci.sh",
+    "package": "tools/package.sh",
+    "release": "semantic-release"
+  },
+  "release": {
+    "branches": [
+      "master"
+    ],
+    "plugins": [
+      [
+        "@semantic-release/github",
+        {
+          "assets": [
+            {
+              "path": "pkg/logdna-s3.zip"
+            }
+          ]
+        }
+      ]
+    ],
+    "extends": "semantic-release-config-logdna"
+  },
+  "eslintConfig": {
+    "root": true,
+    "ignorePatterns": [
+      "node_modules/",
+      "coverage/"
+    ],
+    "extends": [
+      "logdna"
+    ],
+    "parserOptions": {
+      "ecmaVersion": 2019
+    }
+  },
+  "tap": {
+    "esm": false,
+    "ts": false,
+    "jsx": false,
+    "browser": false,
+    "check-coverage": true,
+    "output-file": ".tap",
+    "timeout": 60,
+    "nyc-arg": [
+      "--all=true",
+      "--exclude=test/"
+    ],
+    "coverage-report": [
+      "html",
+      "text",
+      "text-summary",
+      "json",
+      "json-summary"
+    ],
+    "files": [
+      "test/integration",
+      "test/unit"
+    ],
+    "statements": 0,
+    "branches": 0,
+    "functions": 0,
+    "lines": 0
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
```

Comment:
This shouldn't be using `husky`.  It didn't really work for me anyway.

Reply:
Removed!